packages/epo-react-lib/src/styles/globalStyles.ts

```diff
@@ -127,6 +130,9 @@ export const LINE_HEIGHT_BASE = tokens.LINE_HEIGHT_BASE;
 export const PADDING_LARGE = tokens.PADDING_LARGE;
 export const PADDING_MEDIUM = tokens.PADDING_MEDIUM;
 export const PADDING_SMALL = tokens.PADDING_SMALL;
+export const DURATION = tokens.DURATION;
```

Comment:
tokens that were a part of static-investigations and can move over to the library, open to changes on these values